tests/test_sentry_check.py

```diff
@@ -187,6 +187,36 @@ def test_b316(self):
             ),
         ]
 
+    def test_b317(self):
+        bbc = SentryCheck(filename=path("b317.py"))
+        errors = list(bbc.run())
+        assert errors == [
+            (
+                3,
+                0,
+                "B317: Use ``from sentry.utils import json`` instead.",
```

Comment:
I'd change this to a more generic `Use sentry.utils.json instead of the standard library.` since the current message is only analogous to `import json`.

Reply:
Yeah I started templating it, but it just got a little bit hairy.

TBH I think this lint is OKAY